src/routes/community/+page.server.ts

```diff
@@ -44,16 +44,19 @@ export const load = async () => {
     //     return { issues: mockIssues };
     // }
 
-    // fetch issues from github, appwrite/appwrite repo
+    // fetch issues from GitHub, appwrite/appwrite repo
     const response = await fetch('https://api.github.com/repos/appwrite/appwrite/issues');
```

Comment:
https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues you can use the github api to only return open issues 👍🏻 
<img width="584" alt="Bildschirmfoto 2024-08-08 um 15 19 27" src="https://github.com/user-attachments/assets/f6e95e39-2896-4ede-9dd8-6ef0535c71de">